### DISCUSSION TITLE
How to copy text from notification

### DISCUSSION THREAD

**Discussion by johny65** [apache/netbeans#5106] [Q&A] [answered]
Is there a way to copy the text from a notification? I have several notifications indicating a NullPointerException in NetBeans with all the stacktrace, but the text it isn't selectable thus also not copyable. I want to report some issues but I need this text.

**Comment by mbien** [comment, marked as answer]
if you press "show details" you should see a window with the Exception in it. Which you should be able to select `(ctrl+a)` and copy `(ctrl+c)`.

It will also show up in the IDE log `View -> IDE Log` among the rest of the log.

**Comment by johny65** [comment]
I can't find any "show details" button (this is the notification I was talking about):

![Captura desde 2022-12-14 07-08-51](https://user-images.githubusercontent.com/5256776/207567127-dde9dc49-3538-42ac-946b-e3d2ea61556d.png)

And when I click it, this is the text I mentioned it was not copyable:

![Captura desde 2022-12-14 07-12-47](https://user-images.githubusercontent.com/5256776/207567903-9fce227d-1025-4426-bcff-131c3b4579e1.png)

But the `View -> IDE Log` was helpful, there the exception is detailed and I can select and copy text. Thanks!

**Reply by mbien** [reply]
oh. looking at the screenshot it appears as this exception was simply logged as notification. Usually there would be a "show details" link on the right side of this panel if it would be used in the typical way.
